extensions/ql-vscode/package.json

```diff
@@ -798,19 +798,23 @@
       "view/item/context": [
         {
           "command": "codeQLVariantAnalysisRepositories.removeItemContextMenu",
-          "when": "view == codeQLVariantAnalysisRepositories && viewItem =~ /canBeRemoved/"
+          "when": "view == codeQLVariantAnalysisRepositories && viewItem =~ /canBeRemoved/",
+          "group": "2_qlContextMenu@3"
```

Comment:
Can we give the groups names?

Reply:
I guess, I went with naming them similar to the groups we already have.. but there's not really a reason to be cryptic :D

Reply:
Just something a bit less random?

Reply:
Ignore me, just read the docs!